### PR TITLE
Update create-a-dlp-policy-from-a-template.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the steps as we are not able to add M365 groups as members to the role group.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9852

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/create-a-dlp-policy-from-a-template.md [hunk 1]
```diff
@@ -39,13 +39,12 @@ You can also choose the Custom template, which has no default rules, and configu
...
+   
+1. Create a role group on the **Permissions** page of the Microsoft Purview compliance portal. 
 
-3. While creating the role group, use the **Choose Roles** section to add the following role to the role group: **DLP Compliance Management**.
+2. While creating the role group, use the **Choose Roles** section to add the following role to the role group: **DLP Compliance Management**.
     
-4. Use the **Choose Members** section to add the Microsoft 365 group you created before to the role group.
+3. Use the **Choose Members** section to add members to the role group.
 
 Use the **View-Only DLP Compliance Management** role to create role group with view-only privileges to the DLP policies and DLP reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-dlp-policy-from-a-template.md]:
```suggestion
Use the **View-Only DLP Compliance Management** role to create a role group with view-only privileges to the DLP policies and DLP reports.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@chrfox Gentle reminder

**Comment by chrfox** [comment]
article retired

**Pull request closed without merging**